src/components/BorrowerProfile/HeroBackground.vue

```diff
@@ -12,9 +12,9 @@
 			fallback-format="jpg"
 			fit="fill"
 			:alt="contentfulAlt"
-			:width="1024"
-			:height="320"
-			:source-sizes="sourceSizes"
+			:width="statusCard ? 336 : 1024"
```

Comment:
I don't see this working locally, does it work for you?
<img width="645" height="358" alt="image" src="https://github.com/user-attachments/assets/6b0f8854-d1b4-4fcf-9dbe-91085de62b06" />